src/components/cards/two-factor-authentication/SoftwareTokenWorkflow.vue

```diff
@@ -0,0 +1,107 @@
+<script setup lang="ts">
+  import { defineEmits, type Ref } from 'vue';
+  import { useI18n, type Composer } from 'vue-i18n';
+  import { useDisplay } from 'vuetify';
+
+  const { mdAndDown }: { mdAndDown: Ref<boolean> } = useDisplay();
+
+  type Emits = {
+    (event: 'updateHeader', header: string): void;
+    (event: 'onCloseClicked'): void;
+  };
+
+  const emits: Emits = defineEmits<{
+    (event: 'updateHeader', header: string): void;
+    (event: 'onCloseClicked'): void;
+  }>();
+
+  const { t }: Composer = useI18n({ useScope: 'global' });
+
+  type Props = {
+    qrCodeImageBase64: string;
+  };
+
+  defineProps<Props>();
+
+  let workflowStep: number = 0;
+
+  emits('updateHeader', t('admin.person.twoFactorAuthentication.softwareTokenOption'));
```

Comment:
Es wirkt dynamisch, ist aber sehr statisch. Kann man das evtl. mit einer Bedingung im Template gelöst werden?

Reply:
Was genau meinst du mit einer Bedingung im Template?

Reply:
Je mehr ich drüber nachdenke, umso sinnvoller ist diese Variante. Kannst du den emit evtl. noch in einen onMounted Hook wrappen, damit es offensichtlicher ist, dass das on mount passiert?